internal/googlecloud/manufacturer.go

```diff
@@ -0,0 +1,25 @@
+//go:build !(linux || windows)
```

Comment:
And it seems you need the old style build constraints for older Go: https://github.com/grpc/grpc-go/runs/5577974986?check_suite_focus=true

Reply:
Ah, I had completely forgotten about Go 1.16! Added the old-style build constraints.